src/System.Net.WebHeaderCollection/src/System/Net/WebHeaderCollection.cs

```diff
@@ -238,12 +241,13 @@ public override string[] GetValues(int index)
         //     header      - Name of the header.
         // Return Value:
         //     string[] - array of parsed string objects
-        public override string[] GetValues(string header)
+#pragma warning disable CS8610
+        public override string[]? GetValues(string header)
```

Comment:
Let's just disable it around the signature of the method, and could you also add a comment explaining why we need to disable this warning in various places?

Reply:
sure, I assumed proper nesting is always preferred, but yes some methods too long